### ISSUE TITLE
Missing fields in rules description evaluation

### ISSUE THREAD

**Issue by juliancnn** [wazuh/wazuh#7044]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| Any | Analysisd | Manager | Any | Any |

Hi Team,

I have found an error in the option `description`, of the rules.
For example, the next log:

```
39222: *Oct 6 03:32:02.070 mng: %SEC-6-IPACCESSLOGP: list 199 denied tcp 10.0.61.108(1477) -> 10.0.127.20(445), 1 packet
```

With the following rule:
```xml
   <rule id="100005" level="0">
       <if_sid>4100</if_sid>
       <category>firewall</category>
       <description>Firewall rules grouped, dstport: $(dstport)  </description>
  </rule>
```

The output of phase 3 does not save  ``$(dstport)`` variable

```console
╰─# /var/ossec/bin/wazuh-logtest                                                                                                                                                                  130 ↵
Starting wazuh-logtest v4.2.0
Type one log per line

39222: *Oct 6 03:32:02.070 mng: %SEC-6-IPACCESSLOGP: list 199 denied tcp 10.0.61.108(1477) -> 10.0.127.20(445), 1 packe

**Phase 1: Completed pre-decoding.
        full event: '39222: *Oct 6 03:32:02.070 mng: %SEC-6-IPACCESSLOGP: list 199 denied tcp 10.0.61.108(1477) -> 10.0.127.20(445), 1 packe'

**Phase 2: Completed decoding.
        name: 'cisco-ios'
        parent: 'cisco-ios'
        action: 'denied'
        dstip: '10.0.127.20'
        dstport: '445'
        protocol: 'tcp'
        srcip: '10.0.61.108'
        srcport: '1477'

**Phase 3: Completed filtering (rules).
        id: '100005'
        level: '0'
        description: 'Firewall rules grouped, dstport:   ' //<< Missing dstport
        groups: '['local', 'syslog', 'sshd']'
        firedtimes: '1'
        mail: 'False'
```

The following fields are not evaluated either:

- location
- program_name
- hostname
- dstgeoip

The problem seems to be that some static fields are not evaluated to generate the rule description:

https://github.com/wazuh/wazuh/blob/b03b50082493e51b1f9597422b671f3c2907b523/src/analysisd/eventinfo.c#L1025-L1057

Regards,
Julian

**Comment by yanazaeva** [comment]
Hi team,

Reopening this issue too, because the bug with _dstport_ remains in version 4.1.5. I have reopened [this issue](https://github.com/wazuh/wazuh/issues/4352) too and explained everything there. 

Let me know if you need anything else. 

Regards,
Yana.

**Comment by juliancnn** [comment]
I will close the issue as this bug is known in 4.1 and this fix goes into 4.2.
Regards,
Julian.
